src/main/java/elevator/Elevator.java

```diff
@@ -0,0 +1,17 @@
+package elevator;
+
+public interface Elevator{
+ Integer floor();
+ Boolean call();
+ Integer moves();
```

Comment:
@dwaynewade0033 `elevator.moves()` is a good way to get which floor elevator is moving, but can we use existing `Direction` class?
How about this example usage `elevator.direction().to()` or `elevator.direction().floor()`. I'm just trying to find version that works best for readbility.